osgi-dining-hakkers sample broken with scalatest 2.2.3

```
2.3.1 | Error installing boot features
java.lang.Exception: Could not start bundle mvn:org.scalatest/scalatest_2.11/2.2.3 in feature(s) exam-2.6.0: The bundle "org.scalatest_2.2.3 [72]" could not be resolved. Reason: Missing Constraint: Import-Package: org.apache.tools.ant; version="0.0.0"
    at org.apache.karaf.features.internal.FeaturesServiceImpl.installFeatures(FeaturesServiceImpl.java:482)[40:org.apache.karaf.features.core:2.3.1]
    at org.apache.karaf.features.internal.FeaturesServiceImpl$2.run(FeaturesServiceImpl.java:1150)[40:org.apache.karaf.features.core:2.3.1]
Caused by: org.osgi.framework.BundleException: The bundle "org.scalatest_2.2.3 [72]" could not be resolved. Reason: Missing Constraint: Import-Package: org.apache.tools.ant; version="0.0.0"
```

Reported to scalatest: https://github.com/scalatest/scalatest/issues/484

--- a/project/AkkaBuild.scala
+++ b/project/AkkaBuild.scala
@@ -343,8 +343,8 @@ object AkkaBuild extends Build {
     // force publication of artifacts to local maven repo, so latest versions can be used when running maven tests
     compile in Compile <<=
       (publishM2 in actor, publishM2 in testkit, publishM2 in remote, publishM2 in cluster, publishM2 in osgi,
-        publishM2 in slf4j, publishM2 in persistence, compile in Compile) map
-        ((_, _, _, _, _, _, _, c) => c),
+        publishM2 in slf4j, publishM2 in persistence, publishM2 in stream, publishM2 in protobuf, compile in Compile) map
+        ((_, _, _, _, _, _, _, _, _, c) => c),
     test in Test ~= { x => {
       def executeMvnCommands(failureMessage: String, commands: String*) = {
         if ({List("sh", "-c", commands.mkString("cd akka-samples/akka-sample-osgi-dining-hakkers; mvn ", " ", "")) !} != 0)

